src/tado-heating-zone.js

```diff
@@ -112,13 +112,31 @@ function TadoHeatingZone(platform, apiZone) {
     // Stores the contact sensor service
     zone.contactSensorService = contactSensorService;
 
+    // Sets termination variable from zone config
+    var terminationOption;
+    for (var i = 0; i < platform.config.zones.length; i++){
+        if (platform.config.zones[i].zoneId == zone.id){
+            break;
```

Comment:
Shouldn't `terminationOption` bet set here before the `break`, e.g.

```
terminationOption = platform.config.zones[I]. terminationOption;
```

Reply:
Apologies.  Somehow that line disappeared when updating the Github code.